avance del template

Jejiro53 opened the pull request:
corregí y avance el template como lo tenía antes, le hice unos cambios y avance un poco.

Jejiro53 committed:
avance pull request

corregí y avance el template como lo tenía antes, le hice unos cambios y avance un poco.

abraham1669 commented on Proyecto Template parcial 3/Proyecto Template.html:
```diff
@@ -27,23 +27,23 @@
   </div>
     <div class="barra2">
         <font color="#007FFF"><p>About</p></font>
```
No se debe de usar la etiqueta font, ya que esta etiqueta ha sido depreciada en html5, se debe de aplicar una clase a la etiqueta p, ejemplo:
te dejo una referencia del por qué no se debe ocupar [Font etiqueta](https://developer.mozilla.org/es/docs/Web/HTML/Element/font)
```
.paragraph:{
 color: #007fff;
}
```

abraham1669 commented on Proyecto Template parcial 3/Proyecto Template.html:
```diff
@@ -27,23 +27,23 @@
   </div>
-  </div>
-  </div>
-    <div class="container-titulo">
```
Si estás ocupando Bootstrap, no existe la clase container-titulo, es solo container

abraham1669 commented on Proyecto Template parcial 3/Proyecto Template.html:
```diff
@@ -27,23 +27,23 @@
+</div>
+</div>
+ <div class="container-titulo">
        <div class="row2">
```
Si estas ocupando Bootstrap, no existe una clase row2

Jejiro53 committed:
avance del pull request 2

Corregí los errores que el profe me señalo en la clase anterior y avance un poco mas hasta concluir con lo que me correspondía.
